### ISSUE TITLE
Erlang tuples can't be specified

### ISSUE THREAD

**Issue by ajsalminen** [picotrading/jinja2-encoder-macros#3]
There is no way to define a tuple at least via YAML for the jinja2 templates. I didn't see a clear way to add this either as there is only the one sequence type in YAML.

In my case I resorted to adding a way ([see this commit](https://github.com/ajsalminen/config-encoder-macros/commit/4bd318858d6b1f32f8753f900bab00cf67179bf3)) to pass a string as is and without quoting if it's started with "\", a character which cannot exist unescaped in Erlang strings. It of course breaks the 1-to-1 mapping but I feel like it could be useful in other cases as well. Would that be something you'd be willing to incorporate into the template?

**Comment by jtyr** [comment]
This issue was discussed here: https://github.com/picotrading/ansible-rabbitmq/issues/1
Please test it and let me know if it works for you.

**Comment by ajsalminen** [comment]
Yes, that would work. Seems like the same solution applied to a slightly different problem.

Is the ":" a better character choice than the "\" that I proposed? I chose it purely because it cannot be in an erlang string without being escaped.
